float32 gt
uint16 i

---

bool result
